Topic 3/src/test/java/bootcamp/exercises/RomanTest.java

```diff
@@ -0,0 +1,30 @@
+package bootcamp.exercises;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class RomanTest {
+
+	Roman roman= new Roman();
+	
+	@Test
+	public void test1(){
+		assertEquals(roman.toDecimal("V"),5);
+	}
+	
+	@Test
+	public void test2(){
+		assertEquals(roman.toDecimal("I"),1);
+	}
+	
+	/**
+	 * If the roman number is invalid, shoot me an exception.
+	 */
+	@Test(expected= OverflowException.class)
+	public void test3(){
+		roman.toDecimal("AC");
+	}
+
```

Comment:
[High] Think about all possible scenarios. For example, invalid numbers: IIV it's an invalid number.
See other comments about @Before and @After